cantools/database/can/database.py

```diff
@@ -17,6 +17,7 @@
 from .internal_database import InternalDatabase
 from .message import Message
 from .node import Node
+from ..utils import type_sort_signals, sort_signals_by_start_bit, DEFAULT
```

Comment:
I guess it is advisable to rename `DEFAULT` to `dont_sort_signals`, or `sort_signals_default`

Reply:
I used a generic name because I thought this might be useful for other arguments, too. But I don't mind renaming it to `SORT_SIGNALS_DEFAULT`. (Done.) I am surprised that you are suggesting snake case names, though, as constants are [usually](https://www.python.org/dev/peps/pep-0008/#constants) written in upper case. Or does cantools follow a different coding style?

`dont_sort_signals` would not be a fitting name because the default action depends on the file format (when dumping files). Not sorting is the default for kcd files but for dbc files the default is to sort the signals by their start bit in *de*scending order. (I am not particularly happy with this default but I didn't want to change the default behaviour.) I have documented this in the [doc string of `dump_file`](https://github.com/erzoe/cantools/blob/bugfix-dbc/cantools/database/__init__.py#L217).

For loading files the default is to always sort the signals by their start bit in *a*scending order. I have documented this in the [doc string of `load_string`](https://github.com/erzoe/cantools/blob/bugfix-dbc/cantools/database/__init__.py#L314).

If a user does not want sorting they should pass `sort_signals=None`.

Reply:
Thanks for the rename, all is golden now AFAICS...

> I am surprised that you are suggesting snake case names, though, as constants are usually written in upper case. Or does cantools follow a different coding style?

no, you're right. I suppose that this is just a bad old habit of mine ;)